elements/pl-prairiedraw-figure/info.json

```diff
@@ -0,0 +1,14 @@
+{
+    "controller": "pl-prairiedraw-figure.py",
+    "dependencies": {
+        "coreScripts": [
```

Comment:
Are all of these scripts required for PrairieDraw?

Reply:
It seems so:
- `PrairieDraw.js` directly uses
-- `PrairieGeom.js` for some computations
-- `sylvester.js` for representing positions as Vectors
-- `sha1.js` for the TeX-to-image rendering
- `PrairieGeom.js` uses `numeric-1.2.6.min.js` for some calculations
- Since some PD figures are input-responsive (students draw lines on the figures), `underscore.js` and `backbone.js` are needed for responsive figures.